jwst/stpipe/function_wrapper.py

```diff
@@ -29,7 +29,7 @@
 """
 A Step whose only purpose is to wrap an ordinary function.
 """
-from . import Step
+from .step import Step
```

Comment:
This is to get the base `Step` class instead of the renamed `JwstStep` class?

Reply:
Yes, this is to better simulate how the module will behave once it's moved into the stpipe package.